contest/contest_info.py

```diff
@@ -63,10 +63,38 @@ def get_running_contests():
 def get_contestant_list(contest):
```

Comment:
Well, it needs a lot modification QAQ
``get_contestant_problem_submission_list`` and ``get_contestant_problem_submission_list_before_freeze_time`` all use ``Contestant`` type. I should leave the refactor job for @Changron